src/core/entities/customer_entitiy.py

```diff
@@ -0,0 +1,15 @@
+import datetime
+import uuid
+from dataclasses import dataclass
+from typing import Union
+
+from src.core.entities.base_entity import BaseEntity
+
+
+@dataclass
+class Customer(BaseEntity):
+    id: uuid.UUID
```

Comment:
I think id can be move to BaseEntity. Also created_at, and I would add "modified_at", but it is not necessary.

Reply:
Or updated_at -> the name is mingles.